docs.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+# Copyright 2020 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# GitHub Pages
+# ---------------
+# Generate the GitHub pages to deploy
+
+TRAVIS_REPO_SLUG=${1:-google/verible}
```

Comment:
This is a confusingly named variable. For the perspective of the user of this script, who has this in their toplevel checked out repo, everything runs locally and nothing has to to with travis.
The travis repo is just the github path ? So this variable should probably be name `GITHUB_PATH`.

Also looking at all the uses below, it is always assembled together with the full URL, so couldn't we just have
```
  GITHUB_BASE_URL=https://github.com/google/verible
```
and use that below ? (but I suspect you want to set something in travis that utilizes this variable in a different way)